vignettes/Data_retrieval.Rmd

```diff
@@ -498,31 +498,103 @@ mmthds <- mc_methods(aeids)
 
-```{r mc_plots, eval = FALSE}
-# Create Level 4 plot for a single m4id.
-tcplPlot(lvl = 4,  # data level
-         fld = "m4id", # field to query on 
-         val = c(18609966), # value for each field
-                            # values should match their corresponding 'fld'
-         multi = FALSE, # single plot per page - output 6 per page if TRUE
-         verbose = FALSE, # output all details if TRUE
-         output = "console") # output in R console
+Customization of output is possible by specifying parameters, including `output`, `multi`, `fileprefix`, `by`, `verbose`, `nrow`, `ncol`, and `dpi`.  
+
+- The `output` parameter indicates how the plots will be presented. In addition to outputs viewable with the R console, <font face="CMTT10">tcplPlot</font> supports a variety of publication-quality file type options, including raster graphics (PNG, JPG, and TIFF) to retain color quality when printing to photograph and vector graphics (SVG and PDF) to retain image resolution when scaled to large formats.
+
+- The `multi` parameter allows for single or multiple plots per page. `Multi = TRUE` is the default option for PDF outputs, whereas `multi = FALSE` is the only option for other outputs. If using the parameter option `multi = TRUE` output is generated with either 6 plots per page (`verbose = FALSE`) or 4 plots per page (`verbose = TRUE`).
+
+- The `by` parameter indicates how files should be divided, typically by $aeid$ or $spid$.
+
+- The `fileprefix` parameter allows the user to set a custom filename prefix. The standard filename is tcplPlot_sysDate().output (example: tcplPlot_2023_08_02.jpg) or, if `by` parameter is set, tcplPlot_sysDate()_by.output (example: tcplPlot_2023_08_02_aeid_80.pdf). When a `fileprefix` is assigned the default _tcplPlot_ prefix is replaced with the new filename. (example: myplot_2023_08_02_aeid_80.pdf or myplot_2023_08_02.jpg).
+
+- The `verbose` parameter results in a plot that includes a table containing potency and model performance metrics; `verbose = FALSE` is default and the only option in console outputs.  
+
+- The `nrow` parameter specifies the number of rows in the multiple plots per page; this is 2 by default. The `ncol` parameter specifies the number of columns in the multiple plots per page; this is 3 by default. If `verbose = FALSE`, `ncol` is 2. `Nrow` and `ncol` can customize the number of plots included per page.
```

Comment:
So, is there a maximum number of rows and columns that a user can set the output plots to be?  That is, is the default multi plot 6 or 4 depending on the verbose parameter, then if a user wants more than that they can add it OR can users only define fewer plots displayed on the output multi plot?  Basically what is the behavior of using multi, verbose, and nrow/ncol arguments together?  Are there lower and upper bounds on various arguments being used together.  

(Note: There does not need to be a whole lot of information but a concise summary of interdependencies/restrictions on these functional arguments may help users when they get started with tcplPlot.)

Reply:
@madison-feshuk @brown-jason  Is there an upper limit to the nrow or ncol?

Reply:
there is no real limit that we've programmed into the function.

Reply:
It appears that exceeding more than two rows does not reduce the plot size, but rather increases the length of the page. (ncol increases page width)  An upper bounds is forced by ggsave. I got this error when trying to run nrow = 10 or ncol = 9 

```
Error in `ggsave()`:
! Dimensions exceed 50 inches (`height` and `width` are specified in
  inches not pixels).
ℹ If you're sure you want a plot that big, use `limitsize = FALSE`.
```

Is it worth mentioning an upper bounds if there is not a hard coded limit?